source/dpp/runtime/app.d

```diff
@@ -11,22 +11,23 @@ import dpp.from;
 void run(in from!"dpp.runtime.options".Options options) @safe {
     import std.stdio: File;
     import std.exception: enforce;
-    import std.process: execute;
+    import std.process: spawnProcess, wait;
     import std.array: join;
     import std.file: remove;
+    import std.conv : text;
 
     foreach(dppFileName; options.dppFileNames)
         preprocess!File(options, dppFileName, options.toDFileName(dppFileName));
 
     if(options.preprocessOnly) return;
 
     const args = options.dlangCompiler ~ options.dlangCompilerArgs;
-    const res = execute(args);
-    enforce(res.status == 0, "Could not execute `" ~ args.join(" ") ~ "`:\n" ~ res.output);
+    const status = spawnProcess(args).wait();
```

Comment:
Why `spawnProcess.wait` instad of `execute`?

Reply:
So that the compiler output gets out regardless of what d++ thinks. `pragma(msg,...)` needs to get out even on success, as do warnings. Also, you want to see whatever output you might have got before (or during) an infinite loop in the compiler or ctfe, so we can't wait for the compiler to finish before printing, might as well let it do its own printing to `std{out/err}`.

Reply:
Makes sense.